config-htsget-rs/download-config.toml

```diff
@@ -0,0 +1,24 @@
+ticket_server_addr = "0.0.0.0:8080"
+ticket_server_cors_allow_origins = "All"
+
+[[resolvers]]
+regex = "(.*)"
+substitution_string = "$1"
+
+[resolvers.object_type]
+send_encrypted_to_client = true  # the client will receive encrypted files
+# Uncomment the lines below to use predefined key pairs,
+# rather than automatically generating new key pairs for each for each request
+#private_key = "/config/server-rs.sec.pem"
+#public_key = "/config/server-rs.pub.pem"
+
+[resolvers.storage]
+# The url that will be used for the client's url
+response_url = "http://localhost:8443/s3-encrypted/"
+forward_headers = true
+
+[resolvers.storage.endpoints]
+# URL to retrieve an index file from the storage
+index = "http://dockerhost:8443/s3/"
```

Comment:
Why is the index expected to be unencrypted?
Is it just not implemented yet.

Reply:
The index file is not sensitive information, therefore it is served unencrypted. And that is implemented, not sure what you mean

Reply:
I added a short explanation about that [in the readme](https://github.com/GenomicDataInfrastructure/starter-kit-htsget/pull/8/files#r1568437023) (to be committed).